SampleApps/CarthageTest/CarthageTest/ViewController.swift

```diff
@@ -22,7 +22,7 @@ class ViewController: UIViewController {
         let cardClient = BTCardClient(apiClient: apiClient)
         let dataCollector = BTDataCollector(apiClient: apiClient)
         let localPaymentClient = BTLocalPaymentClient(apiClient: apiClient)
-        let payPalClient = BTPayPalClient(apiClient: apiClient)
+        let payPalClient = BTPayPalClient(authorization: "sandbox_9dbg82cq_dcpspy2brwdjr3qn")
```

Comment:
Can we extract the tokenization key into a constant both here and in the `SPMTest` app?